docs/terms.md

```diff
@@ -0,0 +1,31 @@
+# Terms and Conditions
+
+Clangd remote index service is run on a best-effort basis by C++ tools team at
+Google. Google donated the source code and cloud resources to the LLVM project,
```

Comment:
i think we should be wearing our "clangd developer" hats rather than "google c++ tooling team" hats when maintaining this service.

donation of source code sounds weird, i'd just drop that and say `Google donates VM instances in GCP to the LLVM Foundation for hosting this service, and is not tied to it in any other way.`